.circleci/config.yml

```diff
@@ -42,7 +45,7 @@ jobs:
            set -euo pipefail
            echo "Success!"
   release-chart:
-    executor: chart-releaser
+    executor: nix
     steps:
       - build-cr
```

Comment:
Is the `build-cr` step still need, or even working ?
It installs the dependencies, but it should not be required anymore if they are provided by nix.
And I am not sure `apk` is available in the nix environment.